### DISCUSSION TITLE
십자가 질문

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#268] [Q&A]
- [십자가 wiki](https://github.com/SUST22Unity/HumanSurvival/wiki/Document:-%EB%AC%B4%EA%B8%B0#5-%EC%8B%AD%EC%9E%90%EA%B0%80) 참고
- 현재 투사체 속도는 정의되어 있지만, 기본 시작 속도와 기본 가속도는 정의되어 있지 않습니다.
- "처음 시작할 땐 적을 향해 날라가지만 점점 그 반대 방향으로 날아간다."
   - 몇 초 뒤 뒤로 날아간다?
   - 처음 속도에서 점점 가속도를 뺀다?
      - 몇 초마다 뺄까요?
- 회전한다고 기술이 되어 있는 데, 지금 지정된 십자가는 아래 그림을 가집니다. 
![image](https://github.com/SUST22Unity/HumanSurvival/assets/50050003/dc8bc345-63d9-4ef4-8072-f539fa67d762)
- 잔상과 투명도는 어떠한 물체와 상호작용이 되지 않고, 그저 무기 뒤를 따라가는 효과이고 래벨과 관련이 없나요?, 위 그림과 어울릴까요?

**Comment by rapa1130** [comment]
## 기본 시작 속도 기본 가속도 미정의
어떤 말인지 이해하기 어렵습니다. 정의되어 있지 않다는 것이 구체적으로 무슨 말이고 그것이 왜 문제인지 설명 해주시면 답변드리겠습니다.

## 가속도 방식에 대한 질문
문서에서 등가속 운동이라고 표현하였습니다. 방향이 전환되는 시간은 구현과 무관합니다. 해당 시간은 초기 발사 속력과 가속력에 영향을 받습니다. 

자연스러운 움직임을 표현하려면 매우 짧은 시간동안 빼야한다고 생각합니다. 다만, 자세한 기술적인 구현까지 기획자에게 요구하는 것은 적절하지 않다고 생각합니다. 그것은 프로그래머가 판단할 몫이라고 봅니다.

## 잔상 처리
https://assetstore.unity.com/packages/2d/textures-materials/20-evolving-fantasy-rpg-weapons-61204
위 이미지 중 하나로 교체하시는 걸 추천드립니다.

네 잔상은 그저 그래픽효과입니다. 다만 개발 우선순위 및 중요도가 낮다고 봅니다. 따라서 개발 기간이 길어진 상황에서 프로젝트 완수를 앞당겨야 한다는 점을 고려하면 중요하지 않은 잔상효과는 구현하지 않으셔도 괜찮다고 생각합니다.

**Comment by seong0929** [comment]
### 기본 시작 속도 기본 가속도 미정의
투사체 속도의 수치가 정의되어 스크립트 상에 존재하지만, 기본 시작 속도, 기본 가속도는 스크립트 상으로 정의되어 있지 않고, 수치 상으로도 표현되어 있지 않습니다. 다른 무기들과 달리 이 무기에만 존재하는 스탯이죠? 임시적으로 스탯을 지정해 두겠습니다.

### 잔상 처리
단검이랑 같은 이미지 입니다. 잔상 처리는 지금 단계에서 개발을 안 하는 것으로 하겠습니다.
